index.js

```diff
@@ -0,0 +1,45 @@
+const sumAll = (arr) => {
+  let total = 0
+
+  for (let i = 0; i < arr.length; i++) {
+    total += arr[i]
+  }
+
+  return total
+}
+
+const countPosition = (lineup, position) => lineup.filter((lineup) => lineup.position === position).length
+
+const maxGameIds = (arr) => {
+  for (let i = 3; i < arr.length; i++) {
+    if (arr[i] === arr[i - 1] && arr[i] === arr[i - 2] && arr[i] === arr[i - 3]) {
+      return false
+    }
+  }
+
+  return true
+}
```

Comment:
maxGameIds is a variable name, it's a noun. Functions should be action words. `checkMaxGames` or `violatesMaxGames`